aictl/cli/main.py

```diff
@@ -353,6 +371,21 @@ def main():
     )
     t2v_parser.set_defaults(func=t2v)
 
+    t2a_parser = subparsers.add_parser("t2a", help="the text-to-audio subcommand")
```

Comment:
For the model, let's add the model argument here and in the help section specify the 4 options:
```python
    t2a_parser.add_argument(
        "-m", "--model", default="small", help="the MusicGen model to use (options: small, medium, large, melody)"
    )
```